mitodl/hugo-course-publisher
# course banner

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/hugo-course-publisher/issues/311 and https://github.com/mitodl/hugo-course-publisher/issues/312

#### What's this PR do?
This PR adds a new `block` element to the course level `baseof.html` template called "subheader" where content can be loaded below the header.  On the `course_home.html` and `course_section.html` templates, a new partial called `course_banner.html` is referenced.  Right now this only includes the course title with the flat blue background and red bottom border, but this can be altered later specifically for either the course home page or the course section pages or both.

#### How should this be manually tested?
Spin up the site or visit the deploy preview and make sure the course banners look right.

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/99551480-e427bd00-2989-11eb-8a63-17e797e4d7b4.png)
![image](https://user-images.githubusercontent.com/12089658/99551524-f0ac1580-2989-11eb-8375-1f7d8f76104d.png)
